### PR TITLE
move around circuits/contract + enable prettier

### PR DESCRIPTION
**Pull request by nalinbhardwaj**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,16 @@
 {
-    "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "eslint.workingDirectories": ["./client"],
```
Comment (phated) [inline .vscode/settings.json:3]:
```suggestion
  "eslint.workingDirectories": [],
```
?

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+  "solidity.solhintRules": {},
```
Comment (phated) [inline .vscode/settings.json:4]:
```suggestion
```
Does hardhat's plugin use this?

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "mochaExplorer.files": ["eth/test/**/*.{j,t}s"],
+  "mochaExplorer.cwd": "eth",
+  "editor.formatOnSave": true,
+  "prettier.requireConfig": true,
+  "solidity.formatter": "none",
+  "files.insertFinalNewline": true,
+  "rewrap.autoWrap.enabled": true,
```
Comment (phated) [inline .vscode/settings.json:12]:
```suggestion
```
unused

File: .prettierrc [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (phated) [inline .prettierrc:1]: Single ticks are the devil. Shall we use https://www.npmjs.com/package/@projectsophon/prettier-config

**Review by phated — COMMENTED** [review]
Some thoughts

File: .prettierrc (hunk 4 above)
Comment (nalinbhardwaj) [reply .prettierrc:1]: hm im trying to use this in my package.json and it just doesnt work at all, not sure why, maybe my prettier/vscode is old version but it doesnt seem like it

Comment (nalinbhardwaj) [reply .prettierrc:1]: im just going to leave it in repo for now since that seems easier fix

**Pull request closed without merging**